src/components/common/Actions.vue

```diff
@@ -52,8 +53,14 @@ import { LegalServices } from '@/services/'
 import { Navigate } from '@/utils/'
 
 /** This component is only implemented for Correction filings atm. */
-@Component({})
-export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, NameRequestMixin) {
+@Component({
+  mixins: [
+    DateMixin,
+    FilingTemplateMixin,
+    NameRequestMixin
+  ]
+})
+export default class Actions extends Vue {
```

Comment:
There's less red-underlined text in VS Code when I declare mixins this way.

Also I think this syntax is more in line with Vue3 (that we may upgrade to), especially since some decorators (like `Mixins`) go away in Vue3.